sdk/ml/azure-ai-ml/azure/ai/ml/_artifacts/_artifact_utilities.py

```diff
@@ -4,12 +4,16 @@
 
 # pylint: disable=protected-access
 
+import json
 import logging
 import os
 import uuid
 from datetime import datetime, timedelta
 from pathlib import Path
 from typing import Dict, Optional, Tuple, TypeVar, Union
+import requests
```

Comment:
I don't think we're allowed to take a dependency on `requests`

Reply:
Thanks for bringing this to my attention. I talked with Kevin and have refactored to use the HttpPipeline class.